docs/crate-ownership.md

```diff
@@ -0,0 +1,30 @@
+# Create Ownership Rules
+
+*NOTE*: Rules came from disscussion on issue #72
+***
+
+Each crate released on crates.io should have:
+
+1. Two or more owners
+2. Owners should be from different companies
```

Comment:
I don't know if we can realistically enforce it at the moment. Maybe we can soften the lingo here to:

```
Owners should be from different companies, and if that's not possible, from different functional teams.
```

Reply:
I expect that within a couple months we'll start seeing more contributors from other companies and independent and it should become more achievable. I want to say I support this goal, and would like to establish a culture that promotes it, while being realistic about the early-stage shortage of company diversity. :)

Reply:
We have three functional teams working on ICU4X right now: Google i18n, Fuchsia, and Gecko.  Two of them are from the same company, but are distant in the org chart.  I would just replace the word "company" with the word "team" or similar.

Reply:
Risk of echo chamber are higher within a single company, representing a single perspective.

There's a value in having owners be from different companies, once we reach a higher number of companies participating.

Replacing it means that we give up on the goal. I hope we don't have to give up on it, we can just describe the spectrum - stating that we want people from different "organizations" and if that's not possible at least different "teams".
It may be that we'll have a component owned by two people from the same team, just because we have low number of participants yet. But it should be temporary state, and my hope is that this document will serve as a motivator to move us toward first different teams, and eventually different orgs.

Do you see a value in removing the direction toward increased separation?

Reply:
I think that we should read this more in the spirit than actual wording - and as we agreed before, we will bend the rules if we have to. I can soften the statement here, or put a note in the description.

Reply:
PTAL - added NOTE section to the description, to spell it out officially.

Reply:
I came here intending to write something similar to what @nciric already did.  I think that the spirit here is not to have any specific interest group have an overriding majority vote in the interest of fairness.

Perhaps it makes sense to write down this intention clearly, in addition to a proposed mechanism to achieve it.  This way even if we don't pinpoint exactly the correct approach to achieve it, we at least know the intention behind the mechanism.

Reply:
In terms of code owners, I see the main objective as one of redundancy in terms of engineers joining or leaving the project, which is more on the team level than the company level.

In terms of echo chambers, Google i18n (me, elango, cira) and Fuchsia (konstantin, filip) have different perspectives, priorities, and objectives.  If eliminating echo chambers is the goal, then we should look at functional areas, not companies; e.g., Fuchsia and Gecko are probably closer in terms of priorities than Fuchsia and Google i18n.

Altogether, I think all of these axes should be considered.  We want owners to be from a different company, different team, and different functional area, or as many as possible.

Reply:
Thank you for the explanation! I agree!

Reply:
I've tried to capture all of it in the last revision. PTAL.

I'll submit after approval - if there are tweaks to the message please add PR.

Reply:
I preferred the previous revision. This one doesn't capture the gradation of value between "different teams" and "different companies", and makes it look as if it only mattered that both owners are not on the same team.

Maybe in the paragraph, we could have some indication that `where possible, the two owners should be coming from as diverse backgrounds as possible` or `preferably, the two owners should be from different companies. If that's not possible, different orgs, and if that is not possible, different teams". Or something along those lines.

Reply:
Hopefully the final version - added note with suggested meaning. Made bullet somewhat weaker.